include/deal.II/grid/tria.h

```diff
@@ -1628,6 +1628,29 @@ class Triangulation : public Subscriptor
   void set_manifold (const types::manifold_id number);
 
   /**
+   * Set the manifold_id of all of cells and faces to the given
+   * argument.
+   *
+   * @ingroup manifold
+   *
+   * @see
+   * @ref GlossManifoldIndicator "Glossary entry on manifold indicators"
+   */
+  void set_all_manifold_ids (const types::manifold_id &number);
```

Comment:
`types::manifold_id` is just a scalar integer. Pass it by value instead of reference (but keep it `const`). Same for the next function.